.github/workflows/ci.yml

```diff
@@ -5,8 +5,12 @@ on:
     branches:
       - main
```

Comment:
This is only going to run when we merge to `main` - check out [workflow docs](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#running-your-pull_request-workflow-based-on-files-changed-in-a-pull-request) -- would be awesome to have this run on push to a branch like `dev/*`